proxy/src/main/java/com/velocitypowered/proxy/Velocity.java

```diff
@@ -20,6 +20,13 @@
     public static void main(String... args) {
         startTime = System.currentTimeMillis();
         logger.info("Booting up Velocity {}...", Velocity.class.getPackage().getImplementationVersion());
+        logger.info("");
+        logger.info("           _            _ _         \n" +
+                    "__   _____| | ___   ___(_) |_ _   _ \n" +
+                    "\\ \\ / / _ \\ |/ _ \\ / __| | __| | | |\n" +
+                    " \\ V /  __/ | (_) | (__| | |_| |_| |\n" +
+                    "  \\_/ \\___|_|\\___/ \\___|_|\\__|\\__, |\n" +
+                    "                              |___/");
```

Comment:
Doesn't have enough 🏃 - you can do better!

Reply:
More. MORE!

Reply:
Added more velocity